source/doc_versions.py

```diff
@@ -1,9 +1,9 @@
 # The API version strings to display in the documentation
 # COMMON_VERSION is the latest version of any one API, and is considered the overall API version
-COMMON_VERSION = '2016-06-12'
+COMMON_VERSION = '2016-07-01'
```

Comment:
I'm confused why we wouldn't document the error structure for the API, that seems pretty important?  

My preference would be to update the 7/1 docs and publish the version.. [we also really need public facing release notes so that customers can quickly know what has changed in a version, but that's a separate discussion that I am trying to advocate with Nathan].